package.json

```diff
@@ -14,5 +14,10 @@
 		"@ianvs/prettier-plugin-sort-imports": "^4.3.1",
 		"prettier": "^3.3.3",
 		"prettier-plugin-packagejson": "^2.5.3"
+	},
+	"pnpm": {
+		"overrides": {
+			"vite": "https://pkg.pr.new/vite@main"
```

Comment:
Hopefully this is fine, if not we can wait to merge this until the next vite 6 beta release